util/mqtt.go

```diff
@@ -0,0 +1,50 @@
+package util
+
+import (
+	"strconv"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/planetmint/planetmint-go/config"
+	"github.com/planetmint/planetmint-go/x/dao/types"
+)
+
+func SendMqttMessagesToServer(ctx sdk.Context, challenge types.Challenge) {
+	err := sendMqttMessages(challenge)
+	if err != nil {
+		GetAppLogger().Error(ctx, "MQTT error: "+err.Error())
+	}
+}
+
+func sendMqttMessages(challenge types.Challenge) (err error) {
+	conf := config.GetConfig()
+	uri := "tcp://" + conf.MqttDomain + ":" + strconv.FormatInt(int64(conf.MqttPort), 10)
```

Comment:
Better use `uri := fmt.Sprintf(tcp://%s:%d, conf.MqttDomain, conf.MqttPort)` to get rid of the `strconv.FormatInt(int64(...), 10)`